CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+### citus v10.1.2 (August 16, 2021) ###
+
+* Allows more graceful failovers when replication factor > 1
+
+* Fixes a bug that cause partitions to have wrong dist. key after DROP COLUMN
+
+### citus v10.0.5 (August 16, 2021) ###
+
+* Allows more graceful failovers when replication factor > 1
+
+* Fixes a bug that cause partitions to have wrong dist. key after DROP COLUMN
+
+* Improves citus_update_table_statistics and provides distributed deadlock
+  detection
+
+### citus v9.5.7 (August 16, 2021) ###
+
+* Allows more graceful failovers when replication factor > 1
+
+* Fixes a bug that cause partitions to have wrong dist. key after DROP COLUMN
```

Comment:
For all three:

```suggestion
* Fixes a bug that causes partitions to have wrong distribution key after `DROP COLUMN`
```

Reply:
I also made this entry multiline as it passed 80 character limit.